src/components/taskDetails/task-details.module.scss

```diff
@@ -42,9 +42,32 @@ $off-white: #f0f0f0;
     }
 }
 
+.gitButton {
+    @extend .button;
+    background-color: $off-white;
+    max-height: fit-content;
+    min-width: fit-content;
+    padding: 2px;
+    padding-left: 8px;
+    padding-right: 8px;
```

Comment:
Why we have included `padding` and then padding-left /right, only one has to be applied here